database/factories/NewsFactory.php

```diff
@@ -24,7 +25,7 @@ public function definition(): array
     {
         return [
             'Title' => ucwords(fake()->words(2, true)),
-            'user_id' => $this->seedUserByUsername(fake()->userName)->id,
+            'user_id' => User::inRandomOrder()->first()->ID,
```

Comment:
use existing seeded users as news authors

Reply:
Good point, I'll add those

Reply:
Adressed in https://github.com/RetroAchievements/RAWeb/pull/1770/commits/90627af82e73f4be7f15940613d84cd4b2a580c4